api/v1alpha1/fenceagentsremediation_types.go

```diff
@@ -98,6 +99,20 @@ type FenceAgentsRemediationSpec struct {
 	// +kubebuilder:validation:Enum=ResourceDeletion;OutOfServiceTaint
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	RemediationStrategy RemediationStrategyType `json:"remediationStrategy,omitempty"`
+
+	// NodeSecretPrefix is the prefix for the name of the Secret (followed by the node name) which will contain params needed for FAR in order to remediate a particular node.
+	// Using this Secret is optional.
+	// +kubebuilder:default:="fence-agents-credentials-node-"
```

Comment:
Do we want to have CRD validation for this format?

Reply:
I think not, it's just a default value.
I don't think we should limit the user to this format.

Reply:
ok